### PR TITLE
IKASAN-1770 removing the need to have the vaadin 13 compatibility lib…

### PR DESCRIPTION
**Pull request by mick-stewart73**
…rary in place. Moving frontend package management from bower to npm in alignment with the Vaadin migration path.

### CODE REVIEW

File: ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/HospitalCommentsDialog.java [hunk 1]
```diff
@@ -132,7 +132,7 @@ private void init()
         buttonLayout.add(cancel);
 
         this.setWidth("600px");
-        this.setHeight("100%");
+//        this.setHeight("400px");
```
Comment (andrzej-majewski) [inline ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/HospitalCommentsDialog.java:135]: No height set?

File: ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java [hunk 2]
```diff
@@ -46,7 +46,7 @@ public void open(String label)
         layout.setHorizontalComponentAlignment(FlexComponent.Alignment.CENTER, progressBar);
         layout.setSizeFull();
 
-        this.setSizeFull();
+//        this.setSizeFull();
```
Comment (andrzej-majewski) [inline ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java:49]: No size?

File: ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/layout/PersistenceSetupLayout.java [hunk 3]
```diff
@@ -1,13 +1,5 @@
 package org.ikasan.dashboard.ui.layout;
 
-import com.github.appreciated.app.layout.behaviour.Behaviour;
```
Comment (andrzej-majewski) [inline ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/layout/PersistenceSetupLayout.java:3]: Do we need this class at all. I thought we get rid of that screen al together

File: ikasaneip/solr/solr-client/src/main/java/org/ikasan/exclusion/dao/SolrExclusionEventDao.java [hunk 4]
```diff
@@ -59,7 +59,7 @@ public void delete(String moduleName, String flowName, String s)
     public void delete(String errorUri)
     {
         StringBuffer query = new StringBuffer();
-        query.append(ID).append(COLON).append(errorUri);
+        query.append(ID).append(COLON).append("\""+errorUri+"\"");
```
Comment (andrzej-majewski) [inline ikasaneip/solr/solr-client/src/main/java/org/ikasan/exclusion/dao/SolrExclusionEventDao.java:62]: Why is this part of this PR?

**Review by andrzej-majewski — APPROVED** [review]

File: ikasaneip/solr/solr-client/src/main/java/org/ikasan/exclusion/dao/SolrExclusionEventDao.java (hunk 4 above)
Comment (mick-stewart73) [reply ikasaneip/solr/solr-client/src/main/java/org/ikasan/exclusion/dao/SolrExclusionEventDao.java:62]: Part of the broken unit test i saw.

File: ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java (hunk 2 above)
Comment (mick-stewart73) [reply ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java:49]: The change in the way the dialog works automatically sizes to the content. It used to be the case that setting size full did that but it now makes it take the entire screen,

Comment (mick-stewart73) [reply ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java:49]: I'll remove the commented out code.

File: ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/HospitalCommentsDialog.java (hunk 1 above)
Comment (mick-stewart73) [reply ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/HospitalCommentsDialog.java:135]: Once again the dialog behaviour has changed and goes full screen unless explicit.

**Pull request merged**